pkg/webhook/mutation.go

```diff
@@ -33,13 +33,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-type mutationResponse string
-
 const (
-	mutationSkipResponse    mutationResponse = "skip"
-	mutationUnknownResponse mutationResponse = "unknown"
-	mutationAllowResponse   mutationResponse = "allow"
-	mutationErrorResponse   mutationResponse = "error"
+	mutationErrorResponse   requestResponse = "error"
+	mutationSuccessResponse requestResponse = "success"
+	mutationSkipResponse    requestResponse = "skip"
```

Comment:
are we using the "skip" response?

Reply:
Yes, [here](https://github.com/open-policy-agent/gatekeeper/pull/976/files#diff-b51dd886772df2afc559857517988e0b2c597cc944583aa47891f6de293b4f21R123)

Reply:
thanks!